docs/content/version-reporting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Anonymous Version Reporting
```

Comment:
Tempted to rename this `Privacy` or something. `Anonymous Version Reporting` just seems a bit asymmetric compared to the rest of the titles in this section. 

<img width="360" alt="Screen Shot 2020-05-05 at 7 26 40 AM" src="https://user-images.githubusercontent.com/238885/81061147-d903b880-8ea1-11ea-8633-bc9b457dbf1f.png">

WDYT?

Reply:
I agree. Not a fan of the title `Anonymous Version Reporting`. `Privacy` sgtm.